nixos/modules/services/networking/iwd.nix

```diff
@@ -61,6 +61,12 @@ in
           Only one wireless daemon is allowed at the time: networking.wireless.enable and networking.wireless.iwd.enable are mutually exclusive.
         '';
       }
+      {
+        assertion = !(cfg.settings ? General && cfg.settings.General ? UseDefaultInterface);
+        message = ''
+          UseDefaultInterface has moved. Use settings.DriverQuirks.UseDefaultInterface instead.
```

Comment:
This should specify the entire option names:
```
`networking.wireless.iwd.settings.General.UseDefaultInterface` has been deprecated. Use `networking.wireless.iwd.settings.DriverQuirks.UseDefaultInterface` instead.
```

Reply:
:facepalm: thanks; it does make sense to do that.

Reply:
addressed in 8aa7a0765328b318b8d1d23cf5a6934ce60b54f4